tests/test_currency_network_erc165.py

```diff
@@ -35,22 +35,19 @@ def test_supports_interface(currency_network_contract, web3):
     )
 
     # Calculate interface id from function signatures
-    CURRENCY_NETWORK_INTERFACE_ID = hex(
+    CURRENCY_NETWORK_INTERFACE_ID = (
         int.from_bytes(web3.sha3(text="name()")[:4], "big")
         ^ int.from_bytes(web3.sha3(text="symbol()")[:4], "big")
         ^ int.from_bytes(web3.sha3(text="decimals()")[:4], "big")
         ^ int.from_bytes(
-            web3.sha3(text="transfer(address,uint64,uint64,address[],bytes)")[:4], "big"
+            web3.sha3(text="transfer(uint64,uint64,address[],bytes)")[:4], "big"
         )
         ^ int.from_bytes(
-            web3.sha3(
-                text="transferFrom(address,address,uint64,uint64,address[],bytes)"
-            )[:4],
-            "big",
+            web3.sha3(text="transferFrom(uint64,uint64,address[],bytes)")[:4], "big"
         )
         ^ int.from_bytes(web3.sha3(text="balance(address,address)")[:4], "big")
         ^ int.from_bytes(web3.sha3(text="creditline(address,address)")[:4], "big")
-    )
+    ).to_bytes(4, byteorder="big")
```

Comment:
why was that change?

Reply:
Because it was just working out of luck before. if the hex value starts with a `0` it will be dismissed and the result will not be 4 bytes but 3.5 bytes.